src/Features/Core/Portable/CodeCleanup/OrganizeUsingsSettings.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.CodeCleanup
+{
+    internal class OrganizeUsingsSet
+    {
+        public bool IsRemoveUnusedImportEnabled { get; }
+        public bool IsSortImportsEnabled { get; }
+
+        public OrganizeUsingsSet(OptionSet optionSet, string language)
+        {
+            IsRemoveUnusedImportEnabled = optionSet.GetOption(CodeCleanupOptions.RemoveUnusedImports, language);
+            IsSortImportsEnabled = optionSet.GetOption(CodeCleanupOptions.SortImports, language);
+        }
+
+        public OrganizeUsingsSet(bool isRemoveUnusedImportEnabled, bool isSortImportsEnabled)
+        {
+            IsRemoveUnusedImportEnabled = isRemoveUnusedImportEnabled;
+            IsSortImportsEnabled = IsSortImportsEnabled;
```

Comment:
assigning property to itself.

Reply:
i'm going to guess this also indicates insufficient testing for this :)

Reply:
@mavasani this seems like a good candidate for an analyzer. I've seen it missed a decent number of times over the past 15 years.